src/Controller/Back/SignalementFileController.php

```diff
@@ -32,13 +32,17 @@ public function generatePdfSignalement(
         /** @var User $user */
         $user = $this->getUser();
 
-        $message = (new PdfExportMessage())
-            ->setSignalementId($signalement->getId())
-            ->setUserEmail($user->getEmail());
+        if ($signalement->getPhotos()->count() < 21) {
```

Comment:
21 est le résultat du benchmark effectué en locale

Reply:
@sfinx13 Pourquoi 30 puisque dans les logs sentry, on avait `[TYPE_PDF_EXPORT] Unable to send an email: Mail size too large. Maximum mail size limit is 20MB ` ?

Reply:
@hmeneuvrier En moyenne c'est ce que j'avais pour un signalement avec 20 photos (entre 20 et 30 Mo) 
Sinon on aura plus ce message d'erreur car on envoie plus de pièce jointe